docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Release Notes
     * Documentation Changes
     * Testing Changes
         * Add ``pytest-timeout``. All tests that run longer than 6 minutes will fail. :pr:`2374`
+        * Refactored dask tests :pr:`2377`
```

Comment:
omega nit-pick but maybe include why so that when we try to trace this back in the future we know? :P